packages/api/endpoints/ems.js

```diff
@@ -13,6 +13,11 @@ const log = require('@cumulus/common/log');
  * @returns {Promise<Object>} the promise of express response object
  */
 async function post(req, res) {
+  if (!process.env.EmsProductMetadataReport || !process.env.EmsIngestReport
```

Comment:
Why is this checking all of these conditionals here?    Should we consider passing var.ems_deploy in as something like EmsConfigured?

Reply:
A.) I saw that while researching and was glad that `count` is being added to the module level.
B.) I like that idea way better than what I did, so let me implement it that way.

Reply:
Ok added a `EmsConfigured` variable to check if EMS is setup instead of the lambda ARNs